### ISSUE TITLE
automatic clock adjusting no longer shows up

### ISSUE THREAD

**Issue by VorgonBattlecruiser** [hanai3Bi/Switch-OC-Suite#46]
Before version 1.9.0 there was an option to have the game use only as much resources as it needed to reach the max fps.
That feature was really great to save power and now seems to be gone?
At least i cant find or the "misc. settings" page.

**Comment by hanai3Bi** [comment]
are you talking about governor?

1. sys-clk-oc has been reworked based on the new sys-clk, so i need to reimplement functionalities. 
2. because of HOS limitations, governor is bad emulation, and can possibly causes worse performance and worse battery life.

**Comment by VorgonBattlecruiser** [comment]
Yes, thats it! I used to use the governor all the time. Really love that feature. Can you please bring it back?

**Comment by hanai3Bi** [comment]
i've been doing research, but it seems impossible to implement proper governor,

i will just provide old version together with new version and let user choose

**Comment by VorgonBattlecruiser** [comment]
thank you!

**Comment by hanai3Bi** [comment]
i will fix old sys-clk-oc to work with HOS 17, and then post it in discussions

**Comment by thisismyname113** [comment]
hello, is the HOS 17-compatible sys-clk-oc with governor available? I searched for it in discussions but found nothing. I loved the governor feature and I really appreciate your work here sir.
